Documentation/release-notes/4867.md

```diff
@@ -0,0 +1,22 @@
+### `logcat-parse` Improvements
+
+`logcat-parse` now supports parsing the `adb logcat` output format used
+on Android 10 devices.
+
+### Response File Support in Command-line Tools
+
+The following Xamarin.Android tools now support [response files][rf]:
+
+  * `class-parse`
+  * `generator`
+  * `jcw-gen`
+  * `jnimarshalmethod-gen`
+  * `logcat-parse`
+
+
+[rf]: https://docs.microsoft.com/en-us/windows/win32/midl/response-files
+
+#### Android API bindings
+
+Nested types with `protected` visibility within `sealed` types are now
```

Comment:
This looks good. By default, I'll probably end up placing this under the _Issues fixed_ section of the notes, and so I'll put the Java.Interop PR number at the beginning to make it look like the other issues fixed. I'll probably also add a quick appearance of the word "Java" to help front-load the information about which side of the bindings process the change affects:

```
- [Java.Interop GitHub PR 672](https://github.com/xamarin/java.interop/pull/672):
  In bindings projects, nested Java types with ...
```